### DISCUSSION TITLE
proposal to add another extension

### DISCUSSION THREAD

**Discussion by Bwaiz0** [TeamNewPipe/NewPipe#10422] [Questions]
when browsing youtube without logging in, there is no way to filter or block any channel or video from the search results (mark it as "not interested")
there is an extension that adds such functionality to the browser, setting local user rules for filtering

https://github.com/amitbl/blocktube

how about adding it to newpipe?

**Comment by opusforlife2** [comment]
That's a browser extension. It's meant only for web browsers.

What do you mean by "another" extension?

**Reply by Bwaiz0** [reply]
i confused this page with a newpipe fork that has a built-in sponsorblock (browser extension), sorry
